blast_runner.py

```diff
@@ -38,7 +38,7 @@ def run_blast(query, db, out):
                      '-dbtype', "'nucl'", '-out', 'blast_{}'.format(db),
                      '-parse_seqids'])
     subprocess.call(['blastn', '-db', 'blast_{}'.format(db), 'query',
-                     '{}'.format(query), '-out', '{}'.format(out), '-outfmt', '7']) 
+                     '-out', '-outfmt', '7']) 
```

Comment:
Mmm, you still need those arguments, I think. So something like 

```
subprocess.call(['blastn', '-db', 'blast_{}'.format(db), 'query',
                    query, '-out', out, '-outfmt', '7']) 
```

The string `'query'` is different from the variable `query`. Here's some stuff in the python REPL to hopefully clarify:

```
Python 3.8.6 (default, Sep 25 2020, 09:36:53)
[GCC 10.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> print('Im a string')
Im a string
>>> query = 'Im a string'
>>> print(query)
Im a string
>>> print('query')
query
>>> print('Some other string with format: {}'.format(query))
Some other string with format: Im a string
>>> print('Some other string with format: {}'.format('query'))
Some other string with format: query
>>> print('{}'.format(query))
Im a string
>>> print('{}'.format('query'))
query
```